timescale-container/test/sql/import-mtlusovini.etdevice-data.sql

```diff
@@ -0,0 +1,30 @@
+\c quantumleap
```

Comment:
i am not sure i would refer to lusovini in a public repo.

Reply:
got rid of it, thx!